src/index.ts

```diff
@@ -1,44 +1,223 @@
   height: 600,
-  backgroundColor: '#efefef',
+  physics: {
+        default: 'arcade',
+        arcade: {
+            gravity: { y: 300 },
+            debug: false
+        }
+  },
   scene: {
     preload: preload,
-    create: create
+    create: create,
+    update: update
   }
 };
 
 const game = new Phaser.Game(config);
+//END PHASER CONGIG
 
 
-// const imageKey = "logo";
-const cartmanSvgKey = "cartman";
+//GLOBAL VARIABLES
+var platforms; //GLOBAL PLATFORMS
+var stars; //GLOBAL COLLECTABLES
+var bombs; //ENEMIES
+var player; //GLOBAL PLAYER
+var cursors; //GLOBAL CONTROLLER
+var score = 0; //SCORE VALUE
+var gameOver = false; //GAME OVER?
+var scoreText; //SCORE TEXT FOR DISPLAY
+
+
+function preload()
+{
+  //PRELOAD IMAGES
+  this.load.image('sky', 'src/assets/sky.png');
+  this.load.image('ground', 'src/assets/platform.png');
+  this.load.image('star', 'src/assets/star.png');
+  this.load.image('bomb', 'src/assets/bomb.png');
+
+  //PRELOAD SPRITES
+  this.load.spritesheet('dude',
+      'src/assets/dude.png',
+      { frameWidth: 32, frameHeight: 48 }
+  );
 
-function preload() {
-  // this.load.image(imageKey, logoImg);
-  this.load.svg(cartmanSvgKey, 'src/assets/cartman.svg', {
-    width: 416,
-    height: 388,
-  });
 }
 
-function create() {
-  // const logo = this.add.image(400, 150, imageKey);
-  const cartman = this.add.image(400, 150, cartmanSvgKey);
-
-  this.tweens.add({
-    // targets: logo,
-    targets: cartman,
-    y: 450,
-    duration: 2000,
-    ease: "Power2",
-    yoyo: true,
-    loop: -1
+function create()
+{
+  //DRAW BACKGROUND
+  this.add.image(400, 300, 'sky');
+
+
+  //CREATE PLATFORM GROUP PHYSICS
+  platforms = this.physics.add.staticGroup();
+
+  //CREATE BOMB GROUP PHYSICS
+  bombs = this.physics.add.group();
+
+  //CREATE PLATFORMS
+  platforms.create(400, 568, 'ground').setScale(2).refreshBody();
+  platforms.create(600, 400, 'ground');
+  platforms.create(50, 250, 'ground');
+  platforms.create(750, 220, 'ground');
+
+  //CREATE PLAYER
+  player = this.physics.add.sprite(100, 450, 'dude');
+  player.setBounce(0.2);
+  player.setCollideWorldBounds(true);
+
+  //CREATE COLLECTABLES
+  stars = this.physics.add.group({ //CREATE GROUP OF COLLECTABLES
+      key: 'star', //NAME OF SPRITE
+      repeat: 11, //NUMBER OF STARS TO CREATE
+      setXY: { x: 12, y: 0, stepX: 70 } //XY ORIGIN
+  });
+
+  //SET RANDOM BOUNCE TO ADD VARIETY TO COLLECTABLES
+  stars.children.iterate(function (child) {
+      child.setBounceY(Phaser.Math.FloatBetween(0.4, 0.8));
+  });
+
+  //OVERLAP ALL
+  this.physics.add.overlap(player, stars, collectStar, null, this);
+
+  //COLLECTABLES COLLISION
+  this.physics.add.collider(stars, platforms);
+
+  //PLATFORM + PLAYER COLLISION
+  this.physics.add.collider(player, platforms);
+
+  //PLATFORM + BOMB COLLISION (BOUNCE)
+  this.physics.add.collider(bombs, platforms);
+
+  //PLAYER + BOMB COLLISION (DIE)
+  this.physics.add.collider(player, bombs, hitBomb, null);
```

Comment:
Change this line to `this.physics.add.collider(player, bombs, hitBomb, null, this);` and it will solve the issue with `setTint` in the `hitBomb` function.